src/main/java/com/arpnetworking/tsdcore/sinks/PeriodicStatisticsSink.java

```diff
@@ -340,7 +340,9 @@ public void record(final PeriodicData periodicData, final long now) {
                 }
             }
 
-            _age.accumulate(now - periodicData.getStart().plus(periodicData.getPeriod()).toInstant().getMillis());
+            _age.accumulate(
```

Comment:
I don't think we should change the definition of age; certainly I have concerns about its definition varying based on the incoming data (e.g. what happens in a mixed data scenario). We didn't do this in MAD, we should probably just publish a new metric (ideally in both).

Reply:
Done.